homeassistant/components/asuswrt/device_tracker.py

```diff
@@ -81,16 +81,23 @@ def source_type(self) -> str:
     @property
     def extra_state_attributes(self) -> dict[str, any]:
         """Return the attributes."""
-        attrs = {
-            "mac": self._device.mac,
-            "ip_address": self._device.ip_address,
-        }
+        attrs = {}
         if self._device.last_activity:
             attrs["last_time_reachable"] = self._device.last_activity.isoformat(
                 timespec="seconds"
             )
         return attrs
 
+    @property
+    def ip_address(self) -> str:
+        """Return the primary ip address of the device."""
+        return self._device.ip_address
```

Comment:
This will be a breaking change since the old attribute was called `"ip_address` while the new attribute is called `"ip"`.

Reply:
Done